server/app/app.py

```diff
@@ -94,11 +99,9 @@ def dataset_index(url_dataroot=None, dataset=None):
         return common_rest.abort_and_log(
             e.status_code, f"Invalid dataset {dataset}: {e.message}", loglevel=logging.INFO, include_exc_info=True
         )
-
-
-@webbp.errorhandler(RequestException)
-def handle_request_exception(error):
```

Comment:
I think we may want to keep this to ensure that failed requests are logged?

Reply:
This line (for `handle_request_exception` was just moved up - see starting line 70